vcr/vcr.go

```diff
@@ -364,22 +360,23 @@ func (c *vcr) find(ID ssi.URI) (vc.VerifiableCredential, error) {
 	qp := leia.Eq(concept.IDField, ID.String())
 	q := leia.New(qp)
 
-	gIndex := c.globalIndex()
-	docs, err := gIndex.Find(q)
-	if err != nil {
-		return credential, err
-	}
-
-	if len(docs) != 1 {
-		return credential, ErrNotFound
-	}
+	for _, t := range c.registry.Concepts() {
+		docs, err := c.store.Collection(t.CredentialType).Find(q)
```

Comment:
this concept might not have a collection of the given credential type?

Reply:
Then it would be created and it would return no results.